acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/signer/runner/CmdLineParamsDefaultImpl.java

```diff
@@ -180,6 +180,11 @@ private Collection<String> createEth2Args() {
       params.add(Long.toString(signerConfig.getAltairForkEpoch().get()));
     }
 
+    if (signerConfig.getBellatrixForkEpoch().isPresent()) {
+      params.add("--Xnetwork-bellatrix-fork-epoch");
```

Comment:
Wondering why this is an experimental flag? At what point is it no longer experimental?

Reply:
altair/bellatrix fork epochs are known values on various testnets and mainnet. Custom setting of epoch is mainly for minimal network in AT.